src/MOI/MOI_wrapper.jl

```diff
@@ -130,7 +130,7 @@ function _check_ret_optimize(model)
 end
 
 """
-    Optimizer(env::Union{Nothing, Env} = nothing)
+    Optimizer(env::Union{Nothing, Env} = nothing; pass_names::Bool = false)
```

Comment:
Why not an AbstractOptimizerAttribute?

Reply:
Because this is really a CPLEX.jl hack, rather than an option supported by CPLEX.

Reply:
Still, it would make sense to have attributes of the wrapper. I don't consider a hack, but an actual option of the wrapper.

The attribute (with a supports function) path is more generic, for users calling multiple solvers in a more abstract manner.

In fact, Xpress will require this eventually.

Reply:
I agree with @joaquimg. It could be `optimizer_with_attributes(CPLEX.Optimizer, CPLEX.PassNames() => true)`

Reply:
Okay, I'll make a new PR.